mesonbuild/build.py

```diff
@@ -140,9 +187,9 @@ def __init__(self, environment: environment.Environment):
         self.projects_link_args = PerMachine({}, {})  # type: PerMachine[T.Dict[str, T.List[str]]]
         self.tests = []                               # type: T.List['Test']
         self.benchmarks = []                          # type: T.List['Test']
-        self.headers = []
-        self.man = []
-        self.data = []
+        self.headers: T.List[Headers] = []
+        self.man: T.List[Man] = []
+        self.data = []                                # type: T.List[Data]
```

Comment:
This should also be the new syntax now...

Reply:
Ah, good catch. I've fixed that, and updated the Build initializer to use only the new syntax for consistency.